libs/aws/langchain_aws/llms/bedrock.py

```diff
@@ -46,6 +50,33 @@
     "Error: Prompt must alternate between '\n\nHuman:' and '\n\nAssistant:'."
 )
 
+max_retries = int(os.getenv("MAX_RETRIES", 100))
```

Comment:
By default the retries should be turned off so that the default library behaviour is not modified. Also do not introduce env variables into the library. They should be part of the `ChatBedrock` parameters